packages/host/app/services/loader-service.ts

```diff
@@ -6,10 +6,19 @@ import { Loader } from '@cardstack/runtime-common/loader';
 
 import config from '@cardstack/host/config/environment';
 import { shimExternals } from '@cardstack/host/lib/externals';
+import {
+  type RealmSessionResource,
+  getRealmSession,
+} from '@cardstack/host/resources/realm-session';
+import RealmInfoService from '@cardstack/host/services/realm-info-service';
 
 export default class LoaderService extends Service {
   @service declare fastboot: { isFastBoot: boolean };
+  @service declare realmInfoService: RealmInfoService;
+
   @tracked loader = this.makeInstance();
+  // This resources all have the same owner, it's safe to reuse cache.
```

Comment:
Let's add a note on why exactly it's safe to keep the cache here - if I'm not mistaken it's because the owner is the service, which stays around for the whole lifetime of the host app, which in turn assures the resources will not get torn down?